packages/server/lib/controllers/apiAuth.controller.ts

```diff
@@ -292,7 +292,7 @@ class ApiAuthController {
 
         try {
             logCtx = await logContextGetter.create(
-                { id: String(activityLogId), operation: { type: 'auth' }, message: 'Authorization Basic' },
+                { id: String(activityLogId), operation: { type: 'auth', action: 'connect' }, message: 'Authorization Basic' },
```

Comment:
`connect` is a weird action to me. Maybe something a bit clearer, `createConnection` or something?

Reply:
happy to put create_connection

Reply:
Changed